### DISCUSSION TITLE
Megadesk continuous rebooting when trying to flash

### DISCUSSION THREAD

**Discussion by TASSDevon** [gcormier/megadesk#104] [Q&A] [answered]
Hi

I'm trying to flash the serial enabled firmware onto the board (so I can hook up an ESP)
I'm currently using USBasp (https://www.fischl.de/usbasp/) programmer, set to 5V.

Connected all 6 pins, when I connect it to the USB port, I hear the 3 startup beeps continuously.
 When I do the first command:

```
% avrdude -C/Users/devon/Library/Arduino15/packages/ATTinyCore/hardware/avr/1.5.2/avrdude.conf -c usbasp -p t841 -U lfuse:w:0xe2:m

avrdude: set SCK frequency to 187500 Hz
avrdude: AVR device initialized and ready to accept instructions

Reading | ################################################## | 100% 0.00s

avrdude: Device signature = 0x1e9315 (probably t841)
avrdude: reading input file "0xe2"
avrdude: writing lfuse (1 bytes):

Writing | ################################################## | 100% 0.00s

avrdude: 1 bytes of lfuse written
avrdude: verifying lfuse memory against 0xe2:

Reading | ################################################## | 100% 0.00s

avrdude: 1 bytes of lfuse verified

avrdude done.  Thank you.
```

This works but any subsequent command fails, even if I redo this one, only way to be able to successfully do it again is to powercycle. Any ideas what's going on here? I tried feeding it 5V from the programmer instead of from the desk but that made no difference (obviously not at the same time).

**Comment by gcormier** [comment, marked as answer]
You could try lowering the flashing speed to see if that helps? I've never had much luck with the USBAsp.

**Reply by TASSDevon** [reply]
Yep, that did the trick, used -B 32 and it worked flawlessly!
